src/playground/sample_data_ingestion/config/ad_plugin/ad_plugin_index_body.json

```diff
@@ -0,0 +1,13 @@
+{
+    "settings": {
+        "index": {
+            "number_of_shards": 4
+        }
+    },
+    "mappings": {
+        "properties": {
+            "date": {"type": "date"},
```

Comment:
Index bodies support unix time but if I recall, you label them as `"date"` rather than `unix_time`. An example of configuring the index is [here](https://opensearch.org/docs/latest/opensearch/index-templates/#use-component-templates-to-create-an-index-template). `unix_time` is really just a custom data type used by my tooling only and not used for configuring the index creation.